bridge/src/lib.rs

```diff
@@ -3,11 +3,42 @@ extern crate rutie;
 extern crate lazy_static;
 
 mod connection;
+mod reactor;
+mod worker;
 
 use connection::Connection;
-use rutie::{Module, Object, Symbol, RString, Encoding, AnyObject, AnyException, Exception, VM, Thread};
+use once_cell::sync::OnceCell;
+use rutie::{
+    Module, Object, Symbol, RString, Encoding, AnyObject, AnyException, Exception, VM, Thread,
+    NilClass
+};
+use std::sync::Arc;
+use tokio::runtime::{Builder, Runtime};
+use worker::Worker;
+use temporal_sdk_core::{Logger, TelemetryOptionsBuilder};
+
+const RUNTIME_THREAD_COUNT: u8 = 2;
```

Comment:
I'm a bit worried about setting it too high since it's going to be used for client-only use-cases as well. Maybe we'll need to make it configurable as some point